### ISSUE TITLE
Unable to find the Repairdesk invoice connected to the online sale of a tradein item

### ISSUE THREAD

**Issue by JessicaSOS** [Repair-DeskPOS/RepairDesk-BUGS-IMPROVEMENTS#746]
I am unable to find the Repairdesk invoice connected to the online sale of the phone # 10991. I searched the client with several fields (email-phone, name), it does not exist. I looked for the invoice, it does not exist. I tried to go into the product history to find the invoice or the customer, but despite the successful sale, the device is still "in stock" as if no sale had been made. Still, the transaction is approved, we received the payment and the transaction appears correctly in our woo trade.

I went to the repairdesk woo trade settings and it seems to have a problem with syncing with the products. It may be caused by that, I do not know. I checked the API keys and nothing was changed and everything is supposed to work.

Can you help me please.
<img width="913" alt="mail new order" src="https://user-images.githubusercontent.com/40500094/55240863-398ce480-5210-11e9-9ded-a2c6c2236d33.png">
<img width="935" alt="order woo commerce" src="https://user-images.githubusercontent.com/40500094/55240886-44e01000-5210-11e9-9757-dfac9958bffe.png">
<img width="935" alt="error woo commerce" src="https://user-images.githubusercontent.com/40500094/55240906-4ad5f100-5210-11e9-954a-9991e480aaa6.png">

--

French :
Je suis incapable de trouver la facture Repairdesk relié à la vente en ligne du téléphone #10991. J'ai recherché le client avec plusieurs champs (email-téléphone, nom), il n'existe pas. J'ai recherché la facture, elle n'existe pas. J'ai essayé d'aller dans l'historique du produit pour trouver la facture ou le client, mais malgré la vente réussi, l'appareil est encore "in stock" comme si aucune vente n'avait était effectué. Pourtant, la transaction est approuvé, nous avons reçu le paiement et la transaction apparait correctement dans notre woo commerce. 

Je suis allé dans les réglages woo commerce de Repairdesk et il semble avoir un problème de synchronisation avec les produits. C'est peut-être causé par cela, je ne sais pas. J'ai vérifié les clés API et rien n'a été modifié et tout est supposé fonctionner. 

Pouvez-vous m'aider svp.

**Comment by usmanalikhan93** [comment]
Hi @JessicaSOS 

Usman Ali here, Sure thing! Let me take a look to see if I'm getting the same issue as you and I'll circle back to you.

**Comment by usmanalikhan93** [comment]
For internal Use:
https://repairdesk.atlassian.net/browse/REP-1787

**Comment by usmanalikhan93** [comment]
Hi @JessicaSOS 

I hope you are doing well. I have checked this with developer and he said this issue has been fixed now. Can you please check this at your end or let us know if you are facing this issue again ? 

Thanks 
Usman Ali.

**Comment by JessicaSOS** [comment]
I still have the same problem. Now, all sales are no longer tied with repairdesk. Phones remain "in stock" and invoice are not created. Can you help me quickly please.

**Comment by JessicaSOS** [comment]
This problem is not solved, it is reproduced last Friday.

**Comment by usmanalikhan93** [comment]
Let me check this again.

**Comment by JessicaSOS** [comment]
We sold another phone online yesterday and the problem is still there. No trade woo transactions appear in repairdesk and no product adjustments are made either. It's like there's no more sync.

**Comment by usmanalikhan93** [comment]
Hi @JessicaSOS 

I have checked whole scenario again today and spent a time in it. Syncing is done perfectly. Can you please check again ? 

Thanks 
Usman Ali

**Comment by JessicaSOS** [comment]
It is during an online sale that no action takes place. This is not fixed another example: [Order # 28133] (April 13, 2019)

**Comment by usmanalikhan93** [comment]
Hi @JessicaSOS 

I hope you are doing well. Imran (Our CTO) has enabled logs against this, so we can try why invoice is missing in RD. Now if you face again this issue do let us know so we track this from logs and try resolve this ASAP. 

Thanks 
Usman Ali

**Comment by JessicaSOS** [comment]
item #11532
customer are in Repairdesk because she already exist.

**Comment by JessicaSOS** [comment]
New sale still today and the customer does not exist in Repairdesk, the item is still "in stock" and no invoice is produced. In short, no change is made. Can you look at this problem carefully please. It becomes hard to follow for accounting!
Cistomer: 
amanjot singh
79 rue lucie
Montreal QC  J6A 4K1 
5145524027 
driipp25@gmail.com 

Item: # 11457

woo commerce ID: [SOS Phone] Nouvelle vente en ligne (28477) - avril 27, 2019

**Comment by usmanalikhan93** [comment]
Hi @JessicaSOS 

I have informed to relevant developer, he said ok let me debug this today. I will reach you out if I need more info on this.

**Comment by JessicaSOS** [comment]
Thank you, I will wait for your news. Have a good day

**Comment by usmanalikhan93** [comment]
Hi @JessicaSOS 

I hope you are doing well. We have figured out the issue and now it's been resolved. We have also make this invoice correct "woo commerce ID: [SOS Phone] Nouvelle vente en ligne (28477) - avril 27, 2019"
Issue was due to MTS, your one of webhook was not working. Now it's been working fine. Do verify at your end. Let me know if you have any question. 

Thanks 
Usman Ali.

**Comment by JessicaSOS** [comment]
OK, the problem should be solved. Is it possible to enter the following invoices as paid (since they are in woo commerce)?
FL 16093
FL 16027

Thank you

**Comment by usmanalikhan93** [comment]
Hi @JessicaSOS 

I have checked these invoices and it shows source as web. Are you sure you want to make them as paid ?

**Comment by JessicaSOS** [comment]
These are invoices that were paid via woo commerce, but that did not appear as such because of the problem that is now resolved. Maybe you can not do it manually, but if you can, I would like both bills to be listed as paid in repairdesk since they are in the woo trade and paid for by customers .
